app/src/lib/stores/app-store.ts

```diff
@@ -2011,6 +2013,8 @@ export class AppStore extends TypedBaseStore<IAppState> {
     }))
     this.emitUpdate()
 
+    this._hideStashedChanges(repository)
```

Comment:
Can this be moved inside the  `if (selectedSection === RepositorySectionTab.Changes) {`? I don't think it needs to run every time this is called...

Reply:
We actually want to do this when the user chooses to view changes **and** view history, so I think we do need to hide it every time..